checklists/setup-a-circleci-workflow.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Setup a CircleCI Workflow
+category: DevOps
+date: "2020-03-21"
+tags: ['circleci', 'orb']
+description: Steps to configure a CircleCI workflow
+---
+
+- [ ] Make a seperated job for dependencies  
+All installations must be done in a separate job in order to provide artifacts to upcoming ones. Therefore, the next jobs can be run in parallel.
+
+- [ ] Restore previous caches on every job if possible  
+You can read CircleCI [documentation](https://circleci.com/docs/2.0/caching) for further information
```

Comment:
I know the links are different but they will all look alike, maybe we should add "related" to the sentence. 
```suggestion
You can read CircleCI [documentation](https://circleci.com/docs/2.0/caching) for further information
```